wp-content/themes/core/pcss/components/button/_solid.pcss

```diff
@@ -66,6 +109,7 @@ a.c-btn--inverse {
  * ------------------------------------------------------------------------- */
 
 .c-btn--full,
-a.c-btn--full {
+a.c-btn--full,
```

Comment:
Do we use this anywhere in SQ1 at the moment? If not, I'd be inclined to remove it.

Reply:
Removed this.